### PR TITLE
Implement TilesSchema.{fetchTilesByLocations,fetchPlacesbyBbox,fetchTilesByBbox}

### PR DESCRIPTION
**Pull request by c-w**
[project-fortis#52](https://github.com/CatalystCode/project-fortis/issues/52)

### CODE REVIEW

**Review by jcjimenez — APPROVED** [review]
LGTM

File: src/clients/locations/FeatureServiceClient.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ function formatIdsUri(ids) {
 }
 
 function formatBboxUri(north, west, south, east) {
-  return `http://${apiHost}/features/bbox/${north}/${west}/${south}/${east}`;
+  return `http://${apiHost}/features/bbox/${north}/${west}/${south}/${east}?include=bbox`;
```
Comment (erikschlegel) [inline src/clients/locations/FeatureServiceClient.js]: `apiHost` will contain either `http` or `https`

File: src/resolvers-cassandra/Tiles/queries.js [hunk 2]
```diff
@@ -1,40 +1,252 @@
...
 
+const Promise = require('promise');
+const cassandraConnector = require('../../clients/cassandra/CassandraConnector');
+const featureServiceClient = require('../../clients/locations/FeatureServiceClient');
 const withRunTime = require('../shared').withRunTime;
 
 /**
+ * @see http://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#Python
+ */
+function deg2num(lat_deg, lon_deg, zoom) {
```
Comment (erikschlegel) [inline src/resolvers-cassandra/Tiles/queries.js]: I believe this same functionality is already available in `Tile.tileIdFromLatLong` within the `geotile` module

**Review by erikschlegel — APPROVED** [review]
This looks great.

File: src/clients/locations/FeatureServiceClient.js (hunk 1 above)
Comment (c-w) [reply src/clients/locations/FeatureServiceClient.js]: Did this as a separate PR that's now merged; see ad39b8b.

File: src/resolvers-cassandra/Tiles/queries.js (hunk 2 above)
Comment (c-w) [reply src/resolvers-cassandra/Tiles/queries.js]: Cool. Switched to that implementation in ~~e6b737a~~ ~~6fd3c13~~ 36be473.

**Pull request merged**